rye/src/cli/test.rs

```diff
@@ -83,6 +83,9 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
     }
 
     for (idx, project) in projects.iter().enumerate() {
+        if project.workspace().is_some() && project.is_workspace_root() {
```

Comment:
`is_workspace_root` would return true if it was single package so we need to ask for the workspace